.gitlab/ci/sdk-nightly.yml

```diff
@@ -0,0 +1,100 @@
+.sdk-nightly-schedule-rule:
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(schedule)$/ && $DEMISTO_SDK_NIGHTLY'
+
+
+.upload-entities-to-cortex-xsoar: &upload-entities-to-cortex-xsoar
+  - |
+    demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure
+    demisto-sdk upload -i Packs/HelloWorld/TestPlaybooks/playbook-HelloWorld-Test.yml --insecure
+    demisto-sdk upload -i Packs/HelloWorld/Layouts/layoutscontainer-Hello_World_Test_Layout.json --insecure
+    demisto-sdk upload -i Packs/HelloWorld/IncidentFields/incidentfield-Hello_World_Incident_Test.json --insecure
+    demisto-sdk upload -i Packs/HelloWorld/IncidentTypes/incidenttype-Hello_World_Alert_Test.json --insecure
+    demisto-sdk upload -i Packs/HelloWorld/Classifiers/classifier-mapper-incoming-HelloWorldTest.json --insecure
+
+
+#### Commented Out Until Resolve Running Docker in Docker #### 
+# demisto-sdk-nightly:run-unittests-and-lint:
+#   extends:
+#     - .run-unittests-and-lint
+#     - .sdk-nightly-schedule-rule
+#   variables:
+#     DEMISTO_SDK_NIGHTLY: "true"
+#     SLACK_TEST_TYPE: "sdk_unittests"
+#     SLACK_ENV_RESULTS: ${ARTIFACTS_FOLDER}/env_results.json
+#     SLACK_ONLY_IF: "true"
+
+
+demisto-sdk-nightly:run-validations:
+  extends:
+    - .run-validations
+    - .sdk-nightly-schedule-rule
+  variables:
+    DEMISTO_SDK_NIGHTLY: "true"
+    SLACK_TEST_TYPE: "sdk_unittests"
+    SLACK_ENV_RESULTS: ${ARTIFACTS_FOLDER}/env_results.json
+    SLACK_ONLY_IF: "true"
+
+
+demisto-sdk-nightly:create-instance:
+  extends: 
+    - .default-job-settings
+    - .sdk-nightly-schedule-rule
+  variables:
+    DEMISTO_SDK_NIGHTLY: "true"
+    IFRA_ENV_TYPE: "Server Master"
+  cache:
+    policy: pull-push
+  needs: []
+  stage: create-instances
+  script:
+    - !reference [.download-demisto-conf]
+    - |
+      [ -n "${TIME_TO_LIVE}" ] && TTL=${TIME_TO_LIVE} || TTL=300
```

Comment:
Just sets the `TTL` to the value of `TIME_TO_LIVE` if that env var was set (e.g. passing it as parameter in a trigger script) and otherwise setting it to a default of 300.